sheerwater_benchmarking/data/tahmo.py

```diff
@@ -0,0 +1,183 @@
+"""Get Tahmo data."""
+import dask.dataframe as dd
+import xarray as xr
+
+from sheerwater_benchmarking.utils.caching import cacheable
+from sheerwater_benchmarking.utils import get_grid_ds, get_grid, roll_and_agg, apply_mask, clip_region
+from sheerwater_benchmarking.utils.remote import dask_remote
+
+@dask_remote
+@cacheable(data_type='tabular', cache_args=['station_code'], cache=True)
+def tahmo_station(station_code):
+    """Get Tahmo station data."""
+    obs = dd.read_csv(f"gs://sheerwater-datalake/tahmo-data/tahmo_qc_stations_v2/{station_code}.csv",
+                      on_bad_lines="skip")
+
+    # Get a list of columns starting with quality and merge them
+    quality_cols = [col for col in obs.columns if col.startswith('quality')]
+    obs['qual'] = obs[quality_cols].mean(axis=1)
+
+    # Get a list of columns starting with pr and merge them
+    pr_cols = [col for col in obs.columns if col.startswith('pr')]
+    obs['precip'] = obs[pr_cols].mean(axis=1)
```

Comment:
I'm worried this is sus (to average precip sensors) - another option would be something like combine_first, which at least would have a "real" precip measurement from a specific sensor.